ormar/queryset/utils.py

```diff
@@ -17,6 +17,15 @@
     from ormar import Model, BaseField
 
 
+def to_str(val: Union[bytes, str, int]) -> str:
```

Comment:
There already is:

https://github.com/collerek/ormar/blob/8dc05d5cf48469f6285e88ab87d38b57c678976f/ormar/fields/parsers.py#L40-L43

which probably can be re-used

Reply:
All use the `to_str`.